src/main/java/eu/execom/pomodoroTeam/entities/dto/TeamDto.java

```diff
@@ -6,25 +6,10 @@
 
 @Data
 public class TeamDto {
-    @JsonProperty("id")
-    private Long id;
 
     @JsonProperty("name")
     private String name;
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
+    @JsonProperty("id")
+    private Long id;
```

Comment:
Please put the `id` field at the top, since it  should match the way it is done in the `TeamEntity`.